meshdb/browser-tests/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```

Comment:
It looks like we're not super consistent in our directory naming between using hyphens and underscores. Do we want to have a mix of both?

Reply:
Yeah we have been inconsistent here. Part of the problem is that there is an unofficial Go convention of using hyphens, but we also have things like __node_modules/__ which is using underscores.

How about we use hyphens for directory names whenever possible?

Reply:
Yep, I'm down.